go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/libp2p/go-libp2p-nat v0.0.4
 	github.com/libp2p/go-libp2p-net v0.0.2
 	github.com/libp2p/go-libp2p-netutil v0.0.1
-	github.com/libp2p/go-libp2p-peer v0.0.1
+	github.com/libp2p/go-libp2p-peer v0.1.1
```

Comment:
how did we get a minor bump for what seems like a patch release?

Reply:
Two bumps.